src/io.github.fragglet.sdl_sopwith.metainfo.xml

```diff
@@ -0,0 +1,103 @@
+<component type="desktop-application">
+  <id>io.github.fragglet.sdl_sopwith</id>
+  <name>Sopwith</name>
+  <developer_name>Dave L. Clark</developer_name>
+  <developer id="com.davidlclark">
+    <name>Dave L. Clark</name>
+  </developer>
+  <summary>Classic biplane shoot-'em up</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <description>
+    <p>
+      Sopwith is a modern port of the classic 1980s shoot 'em up game of the
+      same name. The game has a World War I aviation theme. The object of
+      Sopwith is to destroy all enemy targets with a somewhat limited airforce.
+    </p>
+    <p>
+      It can be played both in single player and multiplayer mode over a TCP/IP network.
+    </p>
+  </description>
+  <categories>
+    <category>Game</category>
+    <category>ArcadeGame</category>
+  </categories>
+  <url type="homepage">https://fragglet.github.io/sdl-sopwith/</url>
+  <url type="bugtracker">https://github.com/fragglet/sdl-sopwith/issues</url>
+  <url type="faq">https://fragglet.github.io/sdl-sopwith/FAQ.html</url>
+  <url type="contact">https://discord.gg/S2P5wUUzNU</url>
+  <url type="vcs-browser">https://github.com/fragglet/sdl-sopwith</url>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="social-chat">mild</content_attribute>
+  </content_rating>
+  <launchable type="desktop-id">sopwith.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <caption>The player dropping a bomb</caption>
+      <image width="3200" height="1800">https://raw.githubusercontent.com/wjt/flathub/a70639daa7ba79dc26bc2cc7a55099c73e1fdf52/screenshots/1.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="2.5.0" date="2024-04-19">
+      <url>https://github.com/fragglet/sdl-sopwith/releases/tag/sdl-sopwith-2.5.0</url>
+      <description>
+        <p>there is still no editor for making custom levels, but a python library has been added for programmatically generating levels. an example level has been included, the first custom level!</p>
+        <p>objects in custom levels can now be arbitrarily mirrored and rotated.</p>
+        <p>The numeric keypad enter key can now be used to switch into fullscreen mode, and also when entering a hostname (thanks TechKnight).</p>
+        <p>A long-standing bug was fixed where the computer planes would try to take off by flying straight up and then immediately stalling.</p>
+        <p>The score is no longer reset between levels (this was a bug).</p>
+        <p>At end of level, the remaining planes now "tick down" properly.</p>
+        <p>A bug was fixed where the app icon was shown as a black silhouette.</p>
+        <p>The manual page now documents the dashboard layout.</p>
+        <p>The score will no longer be drawn over the medals for very large values.</p>
+      </description>
+    </release>
```

Comment:
Now, you might reasonably ask yourself: can we avoid having to hand-write both this and `NEWS.md`?

In principle, yes. You can use either `appstreamcli news-to-metainfo` to convert from `NEWS.md` to this format; or `appstreamcli metainfo-to-news` to convert in the other direction.

However the plaintext format that appstreamcli expects (in either direction) is slightly different to the current format of `NEWS.md`.